package.json

```diff
@@ -41,6 +42,7 @@
     "@types/mocha": "^2.2.48",
     "@types/node": "^9.4.1",
     "@types/power-assert": "^1.4.29",
+    "chai": "^4.1.2",
```

Comment:
I've added Chai to have more explicit assertions. If you don't like it we can get back to using a default `assert` statement or pick a new library.

Reply:
This is good, the power assert that I have been using is sometimes a pain, so yeah `chai` works for me :)